pom.xml

```diff
@@ -324,7 +324,7 @@
         <version.org.jboss.mod_cluster>1.3.9.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.8.1.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.1.Final</version.org.jboss.openjdk-orb>
-        <version.org.jboss.resteasy>3.5.1.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>3.6.0.CR1</version.org.jboss.resteasy>
```

Comment:
We probably need some clarification about the rules, but we're supposed to be always final for dependencies.

Reply:
As I've written above, I'm going to update the PR here to use 3.6.0.Final before the PR is actually merged.